Packages/com.unity.inputsystem/InputSystem/Plugins/EnhancedTouch/TouchSimulation.cs

```diff
@@ -264,6 +273,28 @@ protected void OnEnable()
 
             InputSystem.onDeviceChange += m_OnDeviceChange;
             InputSystem.onEvent += m_OnEvent;
+            
+            // In case there's a PlayerInput component in the scene, we want to pair the simulated touchscreen to
+            // the PlayerInput user. The touchscreen device does not queue events so we need to pair it to the user.
+            m_PlayerInput = PlayerInput.s_AllActivePlayersCount > 0 ? PlayerInput.s_AllActivePlayers[0] : null; 
+            if (m_PlayerInput)
+            {
+                TryPerformUserPairingOfSimulatedTouchscreen();
+                InputUser.onChange += PairSimulatedTouchscreenOnChange;
+            }
+        }
+        private void PairSimulatedTouchscreenOnChange(InputUser user, InputUserChange change, InputDevice device)
```

Comment:
Is the reported issue also an issue when not using control schemes or does it only surface when the control schemes are present? Basically what would the intended behavior be in case a user was using a real mouse and a real touch screen at the same time? I would expect ping ponging effects on pointer. So maybe this is different since one want to avoid that and the current system do not support reasoning about sources of synthesised events...

Reply:
>Is the reported issue also an issue when not using control schemes or does it only surface when the control schemes are present?

I didn't test this without control schemes, as they essentially control the PlayerInput use device pairing mechanism (which uses input events). And the problem is pairing mechanism was not taking into account the simulated touchscreen, because it doesn't generate input events.

> Basically what would the intended behavior be in case a user was using a real mouse and a real touch screen at the same time? I would expect ping ponging effects on pointer

Yes that would be the behavior. But for this case, mouse (or pen) doesn't exist as its driving the simulated touchscreen. Also the ping pong only ocurrs if "Auto-switch" in PlayerInput is selected. Unless I'm misunderstanding something.

I'm confused with what you mean with this. Can you clarify?
> So maybe this is different since one want to avoid that and the current system do not support reasoning about sources of synthesised events...